src/components/TagSet/TagSet.mdx

```diff
@@ -102,34 +102,75 @@ Notes:
 - The `isDismissible` prop is optional and the `onClick` prop is required for
```

Comment:
The `onClick` prop for the `"filter"` variant has a `?`, so it is optional.

Reply:
This was always optional. The `onClick` prop is only applicable when `isDismissible` is true.

Reply:
But you say `onClick` is required.  Did you mean it is required when `isDismissible` is true?

Reply:
Ah, I see. This wasn't updated in this pr but I'll update it.

Reply:
Oh, you'r right.  Sorry about that.  Thanks for making the update.